docs/contributing.rst

```diff
@@ -16,7 +16,9 @@ ________
 To contribute to the cosmos project:
 
 #. Please create a `GitHub Issue <https://github.com/astronomer/astronomer-cosmos/issues>`_ describing your contribution
-#. Open a feature branch off of the ``main`` branch and create a Pull Request into the ``main`` branch from your feature branch
+#. Fork the repository and clone your fork locally.
+#. Open a feature branch off of the main branch in your fork
```

Comment:
Would it be worth linking some official docs on how to fork, such as https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo?

Reply:
Pushed a fix in d432c1d58e78fee4b51e69ef0e0045672be324be